server/src/main/java/org/apache/gravitino/server/web/rest/FilesetOperations.java

```diff
@@ -202,20 +200,30 @@ public Response createFileset(
   @ResponseMetered(name = "load-fileset", absolute = true)
   @AuthorizationExpression(
       expression =
-          "METALAKE::READ_FILESET || CATALOG::READ_FILESET "
-              + "|| SCHEMA::READ_FILESET || FILESET::READ_FILESET "
-              + "|| METALAKE::WRITE_FILESET || CATALOG::WRITE_FILESET "
-              + "|| SCHEMA::WRITE_FILESET || FILESET::WRITE_FILESET "
-              + "|| METALAKE::OWNER || CATALOG::OWNER "
-              + "|| SCHEMA::OWNER || FILESET::OWNER",
+          "ANY(OWNER, METALAKE, CATALOG, SCHEMA, FILESET) ||"
+              + "("
+              + "   ANY(USE_CATALOG, METALAKE, CATALOG, SCHEMA) && "
+              + "   ( ANY(READ_FILESET, METALAKE, CATALOG, SCHEMA, FILESET) || ANY(WRITE_FILESET, METALAKE, CATALOG, SCHEMA, FILESET))"
```

Comment:
please add a ANY_WRITE_FILESET in org.apache.gravitino.server.authorization.expression.AuthorizationExpressionConverter such as ANY_USE_CATALOG